tests/test-cases/Group1-Docker-Commands/1-02-Docker-Pull.robot

```diff
@@ -119,3 +119,16 @@ Pull image by multiple tags
     ${id1}=  Get From List  ${words1}  2
     ${id2}=  Get From List  ${words2}  2
     Should Be Equal  ${id1}  ${id2}
+
+Issue docker pull on digest outputted by previous pull
+    ${rc}  Run And Return Rc  docker %{VCH-PARAMS} rmi busybox
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull busybox | grep Digest | awk '{print $2}'
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Be Empty  ${output}
+    ${rc}  Run And Return Rc  docker %{VCH-PARAMS} rmi busybox
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull busybox@${output} | grep Status | awk '{print $2}'
```

Comment:
We really don't need to grep and awk this one, it makes it harder to debug failures as you throw away all the output.  In this case you can really just keep all the output and check RC and entire output contains Downloaded